optuna/samplers/_cmaes.py

```diff
@@ -767,6 +767,14 @@ def _get_trials(self, study: "optuna.Study") -> List[FrozenTrial]:
                 _, value = max(t.intermediate_values.items())
                 if value is None:
                     continue
+
+                if not isinstance(value, float):
+                    warnings.warn(
+                        "This sampler doesn't support multiple intermediate values. "
+                        "Skip this trial."
+                    )
```

Comment:
I noticed that this warning message is repeatedly being displayed. Would it be possible to limit the output of the message to at most once per trial, or alternatively, consider not displaying the message at all? I apologize for any confusion caused.

Reply:
I see. For now, I deleted sampler-related warnings once and for all.